lte/gateway/docker/readme.md

```diff
@@ -1,170 +1,29 @@
- ---> Running in 46435f4fbad2
-Selecting previously unselected package python3-aioeventlet.
-(Reading database ... 47574 files and directories currently installed.)
-Preparing to unpack .../python3-aioeventlet_0.5.1-2focal_amd64.deb ...
-Unpacking python3-aioeventlet (0.5.1-2) ...
-Setting up python3-aioeventlet (0.5.1-2) ...
-Removing intermediate container 46435f4fbad2
- ---> 08ed545b9000
-
-Successfully built 08ed545b9000
-Successfully tagged agw_gateway_python:latest
-sdti-build1:~/magma/lte/gateway/docker #
-```
-* Check images on local host
-* ctr-build:~/magma-ctr/magma/lte/gateway/docker #docker images
-
-```
-    REPOSITORY           TAG       IMAGE ID       CREATED          SIZE
-    agw_gateway_c        latest    755b206a9698   3 minutes ago    1.29GB
-    <none>               <none>    5163a21390e8   6 minutes ago    4.41GB
-    agw_gateway_python   latest    d61fcb3ed86e   26 minutes ago   894MB
-    <none>               <none>    6cb8b28381ce   29 minutes ago   1.71GB
-    <none>               <none>    06c85a31b481   2 hours ago      4.41GB
-    <none>               <none>    b6e324a3c73a   3 hours ago      1.71GB
-    ubuntu               focal     ba6acccedd29   3 weeks ago      72.8MB
-```
-
-* Ensure repositories are created on dockerhub
-* Tag and Push images to docker hub
-
-```
-ctr-build:~/magma-ctr/magma/lte/gateway/docker #docker image tag agw_gateway_python:latest arunuke/agw_gateway_python:9Nov
-ctr-build:~/magma-ctr/magma/lte/gateway/docker #docker image tag agw_gateway_c:latest arunuke/agw_gateway_c:9Nov
-ctr-build:~/magma-ctr/magma/lte/gateway/docker #docker image push arunuke/agw_gateway_c:9Nov
-ctr-build:~/magma-ctr/magma/lte/gateway/docker #docker image push arunuke/agw_python:9Nov
-```
-
-* Use the Base Cloudstrapper image with an expanded disk size (64G at least) to create Test Container
-
-```
-ansible-playbook agw-provision.yaml --tags createGw -e '@~/magma-master/sdti-ctr1.yaml'
-
-evsrv-tokyo:~/magma-master/magma/experimental/cloudstrapper/playbooks #cat ~/magma-master/sdti-ctr1.yaml
----
-#Setting AGW AMI and Cloudstrapper AMI to expanded Cloudstrapper image Ubuntu to allow deploy
-dirLocalInventory: ~/magma-master
-awsAgwAmi: ami-03bc7ef7f3b70f77b
-awsCloudstrapperAmi: ami-03bc7ef7f3b70f77b
-awsAgwRegion: ap-northeast-1
-keyHost: keyMagmaHostCharlie
-idSite: SDTI
-idGw: sdti-ctr1
-awsInstanceType: t3.large
-```
-
-* Prepare the host
-    * Install ifupdown
-    * Unlink `/etc/resolv.conf` and create a new one with 8.8.8.8 entry
-    * Create `/var/opt/magma/certs `and add rootCA.pem to that folder with permissions 400
-    * Copy `agw_install_docker.bash` and run script to prepare the host
+* Build the docker images and push to a registry
+* Look at ../../../experimental/cloudstrapper/README.md for an AWS setup
+    * If you are only interested in running the AGW, you only need steps 1 and 5.1
```

Comment:
you should explain briefly what the results of the steps are an why the are run.
For example 1. "Prepare the aws connections and keys for the hosts."
and so on